Lesson00/Homework/ToDo/Views/Home/About.cshtml

```diff
@@ -1,6 +1,19 @@
 ﻿@{
     ViewData["Title"] = "About";
 }
+
+@{
```

Comment:
Good job! There is another way you could have done this though without repeating yourself. Did you happen to run across anything about the _ViewStart.cshtml in the readings and how it might help us in this scenario?